src/services/tag.service.tsx

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from 'injection-js';
+import { Tag } from '../dtos/tag.dto';
+
+@Injectable()
```

Comment:
You can remove the inject relating things. The `Injectable` annotation is an Angular construct to handle dependency injection.